### PR TITLE
INS-20756-opensearch-1.3.5-terraform-doc

### PR DESCRIPTION
**Pull request by yiwei-wu-instaclustr**

### CODE REVIEW

File: examples/main.tf [hunk 1]
```diff
@@ -275,7 +275,7 @@ resource "instaclustr_cluster" "example-opensearch" {
 
   bundle {
     bundle = "OPENSEARCH"
-    version = "1.3.4"
+    version = "1.3.5"
     options = {
       dedicated_master_nodes = true,
       master_node_size = "SRH-DEV-t4g.small-5",
```
Comment (iclonghoang) [inline examples/main.tf:281]: can you also test and fix this example as it's incorrect, i.e. dedicated master node size is wrong

Comment (yiwei-wu-instaclustr) [reply examples/main.tf]: Thanks Long, updated

Comment (iclonghoang) [reply examples/main.tf]: all good
you can have this high ticket resolved with this fix
https://instaclustr.atlassian.net/browse/INS-20681

**Comment by iclonghoang** [comment]
also needs version bump for releasing

**Comment by yiwei-wu-instaclustr** [comment]
Tests pending OpenSearch 1.3.5 release

**Review by iclonghoang — APPROVED** [review]

File: examples/main.tf [hunk 2]
```diff
@@ -520,7 +520,7 @@ resource "instaclustr_cluster" "example-cadenceav-opensearch" {
 
   bundle {
     bundle = "OPENSEARCH"
-    version = "1.3.4"
+    version = "1.3.5"
     options = {
```
Comment (iclonghoang) [inline examples/main.tf:524]: this seems need fixing node sizes
can you also run it to test?

**Review by iclonghoang — APPROVED** [review]

**Review by TianyaoJiang — APPROVED** [review]

File: examples/main.tf (hunk 2 above)
Comment (yiwei-wu-instaclustr) [reply examples/main.tf:524]: That's right, updated. Testing provision Cadence failed at the cassandra version, and now checking.

**Review by TianyaoJiang — APPROVED** [review]

**Review by Mitchel-Stig-Instaclustr — APPROVED** [review]

**Pull request merged**